src/views/newConnection.tsx

```diff
@@ -43,64 +45,92 @@ export default class newConnectionTemplates {
     return (
       <>
         <form id="new-connection-form" hx-post="/connection/new/submit" hx-swap="outerHTML">
-          <div>
-            <this.stepper formStage={params.formStage} />
-            <input
-              id="company-number-input"
-              class="new connection"
-              name="companyNumber"
-              placeholder="Company House Number"
-              required
-              hx-get="/connection/new/verify-company"
-              hx-trigger="keyup changed delay:200ms, change"
-              hx-target="#text-box-target"
-              hx-select="#text-box-target"
-              pattern={companyNumberRegex.source}
-              minlength={8}
-              maxlength={8}
-              oninput="this.reportValidity()"
-              value={params.companyNumber}
-              type={`${showForm ? 'text' : 'hidden'}`}
-            ></input>
-            <input
-              id="email"
-              placeholder="Connection's Email Address"
-              name="email"
-              value={params.email}
-              type={`${showForm ? 'email' : 'hidden'}`}
-            ></input>
-            <div id="confirmation-page" style={{ display: showConfirmation ? 'block' : 'none' }}>
-              <p>Please confirm the details of the connection before sending</p>
-              <p>
-                {Html.escapeHtml(
-                  `Company House Number: ${params.targetBox.status === 'success' && params.targetBox.company.company_number}`
-                )}
-              </p>
-              <p>{Html.escapeHtml(`Email Address: ${params.targetBox.status === 'success' && params.email}`)}</p>
+          <this.stepper formStage={params.formStage} />
+          <div class="align-horizontally">
+            <div>
+              <div class="align-vertically ">
```

Comment:
This is quite the div soup. I migth need you to walk me through why we need this.